tests/test_cases/getcwd.c

```diff
@@ -1,16 +1,16 @@
-#include <unistd.h>
-#include <stdio.h>
-
-int main() {
-  int buffersize = 256;
-  char cwd[buffersize];
-
-  char* result = getcwd(cwd, sizeof(cwd));
-
-  if (result == NULL)
-    perror("getcwd() error");
-  else
-    printf("current working directory is: %s :: %s\n", cwd, result);
-
-  return 0;
-}
+#include <unistd.h>
```

Comment:
I think caglars commit also added a getcwd test? so this may be redundant. I'd go with his if thats true?

Reply:
As far as I can see, there is no change in the file with this MR so this should not be a problem. I am not sure why git is treating this as a changed file.